<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rb2D">
             <property name="text">
              <string>2D</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb2Dseries">
             <property name="text">
              <string>2D series</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb3D">
             <property name="text">
              <string>3D</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/curves.xpm</normaloff>:/icons/icons/curves.xpm</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool2">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/poly.xpm</normaloff>:/icons/icons/poly.xpm</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="R0">
             <property name="text">
              <string>R0</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R1">
             <property name="text">
              <string>R1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R2">
             <property name="text">
              <string>R2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R3">
             <property name="text">
              <string>R3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R4">
             <property name="text">
              <string>R4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R5">
             <property name="text">
              <string>R5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R6">
             <property name="text">
              <string>R6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R7">
             <property name="text">
              <string>R7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R8">
             <property name="text">
              <string>R8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="R9">
             <property name="text">
              <string>R9</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QSlider" name="SlicePos">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export Image</string>
     </property>
     <addaction name="actionMAT4"/>
     <addaction name="actionBitmap"/>
     <addaction name="actionBitmap_series"/>
    </widget>
    <addaction name="actionBrowse"/>
    <addaction name="action_Open"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Files"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionScreenshot"/>
    <addaction name="actionScreenshot_on_ROI"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>438</width>
     <height>584</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Regions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <layout class="QGridLayout" name="actionLayout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>arg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="arg"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Scripts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="scriptBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="applyScript">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPlainTextEdit" name="batchActions2"/>
         </item>
         <item>
          <widget class="QToolButton" name="applyBatch2">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPlainTextEdit" name="batchActions"/>
         </item>
         <item>
          <widget class="QToolButton" name="applyBatch">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="batchActions3"/>
     </item>
     <item>
      <widget class="QToolButton" name="applyBatch3">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>222</width>
     <height>159</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>350</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Region Display</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="showNone">
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="showBoundary">
         <property name="text">
          <string>Boundary</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="showMask">
         <property name="text">
          <string>Mask</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="showBlend">
         <property name="text">
          <string>Blend</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="display_ratio">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="min_value"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="max_value"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="style">
       <item>
        <property name="text">
         <string>grayscle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>color</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>174</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="clearlog">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMAT4">
   <property name="text">
    <string>MAT4...</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse...</string>
   </property>
  </action>
  <action name="actionBitmap">
   <property name="text">
    <string>Bitmap</string>
   </property>
  </action>
  <action name="actionBitmap_series">
   <property name="text">
    <string>Bitmap series</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Copy whole view</string>
   </property>
  </action>
  <action name="actionScreenshot_on_ROI">
   <property name="text">
    <string>Copy ROI &amp;View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
